infra/modules/monitoring/main.tf

```diff
@@ -0,0 +1,69 @@
+# Create SNS topic for all email and external incident management tools notifications
+
+resource "aws_sns_topic" "this" {
+  name = "${var.service_name}-monitoring-notifications"
```

Comment:
nit: "notifications" is a bit redundant since the N in SNS already means notifications what about just `${var.service_name}-monitoring`?